Projects/Server/Buffers/SpanWriter.cs

```diff
@@ -438,7 +438,7 @@ public int Seek(int offset, SeekOrigin origin)
             _                  => offset // Begin
         });
 
-        if (newPosition >= _buffer.Length)
+        if (newPosition > _buffer.Length)
```

Comment:
This was causing seeks to the end of the spanwriter (for writing packet lengths), to cause the buffer to grow unnecessarily.